Makefile

```diff
@@ -217,12 +217,12 @@ tagAndPushToDockerHub: tag
 	docker push ${IMAGE_REPO}/natssync-tests:latest
 	docker push ${IMAGE_REPO}/httpproxy-server:${IMAGE_TAG}
 	docker push ${IMAGE_REPO}/httpproxy-server:latest
+#   httpproxy_server is depricated use httpproxy-server instead 
```

Comment:
these 3 were recently removed, not needed

Reply:
thanks I had trouble during the merge

Reply:
I will replace this with the existing make file from main